src/share/util/mct_mod.F90

```diff
@@ -98,6 +98,7 @@ module mct_mod
    use m_GlobalSegMap       ,only: mct_gsMap_gstorage     => GlobalStorage
    use m_GlobalSegMap       ,only: mct_gsMap_ngseg        => ngseg
    use m_GlobalSegMap       ,only: mct_gsMap_nlseg        => nlseg
+   use m_GlobalSegMap       ,only: mct_gsMap_OP           => OrderedPoints
```

Comment:
This line was removed because it was a redefinition of mct_gsMap_orderedPoints on line 106.  I believe you need to make a change in the land model from mct_geMap_OP to mct_gsMap_orderedPoints